sourcegraph/sourcegraph
# bext: fix extensions decorations on GitHub

Closes https://github.com/codecov/sourcegraph-codecov/issues/86

## Test plan
- `sg run bext`
- ensure Codecov extension is enabled
- go to any repo integrated with Codecov (e.g. [notaryproject/notary](https://github.com/notaryproject/notary))
- navigate through the file system without reloading the page and ensure that extensions decorations are rendered each time (beware that not all files have code coverage, but most of [these](https://github.com/notaryproject/notary/tree/master/server/storage) have)



## App preview:

- [Web](https://sg-web-taras-yemets-bext-fix-decorations.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-gostkzxmzj.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
